### ISSUE TITLE
Tooltips not showing up

### ISSUE THREAD

**Issue by nwolverson** [sbrink/vscode-elm#11]
Platform: Win7
Extension version: 0.2.0

When I hover over a token I don't see any type information, instead in the dev console I see (once per attempt):

```
[Plugin Host] Object {code: "EPIPE", errno: "EPIPE", syscall: "write"}
(anonymous function) @ workbench.main.js:1602
emitTwo @ events.js:87emit @ events.js:172
handleMessage @ internal/child_process.js:685
channel.onread @ internal/child_process.js:440
```

**Comment by nwolverson** [comment]
The error is raised from `cp.execFile` in `ElmHoverProvider` and not caught.

I didn't have `elm-oracle` installed globally, while `elm-oracle` seems to be installed locally within the extension that's not being exec'd. Secondly it seems that on windows the `execFile` must be called with `elm-oracle.cmd`. If I change to `.cmd` and install globally, I see tooltips.

**Comment by Krzysztof-Cieslak** [comment]
Fixed in `0.2.2`

**Comment by nwolverson** [comment]
Much better cheers :)
